File: 1328-break-a-palindrome/1328-break-a-palindrome.py
class Solution:
    def breakPalindrome(self, palindrome: str) -> str:
        if len(palindrome) == 1:
            return ""
        
        for i in range(len(palindrome)):
            if palindrome[i] != "a":
                for char in palindrome[i+1:]:
                    if char != "a":
                        return palindrome[:i] + "a" + palindrome[i+1:]
                return palindrome[:-1] + "b"
        
        return palindrome[:-1] + "b"

File: 1328-break-a-palindrome/1328-break-a-palindrome.js
/**
 * @param {string} palindrome
 * @return {string}
 */
var breakPalindrome = function(palindrome) {
    if (palindrome.length === 1) return "";
    
    for (i = 0; i < palindrome.length; i++) {
        if (palindrome[i] != "a") {
            for (j = i+1; j < palindrome.length; j++) {
                if (palindrome[j] != "a") {
                    return palindrome.slice(0, i) + "a" + palindrome.slice(i+1);
                }
            }

            return palindrome.slice(0, palindrome.length-1) + "b";
        }
    }

    return palindrome.slice(0, palindrome.length-1) + "b";
};